### ISSUE TITLE
How do I have it run in startup and hang around in system tray?

### ISSUE THREAD

**Issue by Blessedwasnotfat** [Pizzabelly/EasyRP#136]
I'd like it to run when I turn my PC on, but not be visible in the taskbar.

Any way I can do this?
I have a shortcut for the EasyRP so I don't have to manually find the exe file.

**Comment by bhavyagosai** [comment]
Yeah you can do that by writing an AHK script. Just download [AHK](https://www.autohotkey.com/) and then create an .ahk file and write the code to run a specific application "in hidden mode" upon the script run. The script code would be something like this:  
(FOLLOW EACH AND EVERY SPACE PROPERLY AND FILL IN THE PARTS WITH <> NOTATIONS)

```
Run, <directory of your .exe file>, ,Hide
WinWait, ahk_exe <<exact name of the exe file>.exe>
ExitApp
```
After completing your script you can make it autorun everytime upon booting up your PC by putting its shortcut in the RUN=>shell:startup=>folder. Check [this](https://www.digitalcitizen.life/enable-apps-run-windows-10-startup/) if you don't know how to do this.

If you face any problems lemme know. Also if you want your EasyRP to work in a better way then check out this [repo!](https://github.com/bBSempai/EasyRP)

**Comment by jakiisheep** [comment]
> 
> 
> Yeah you can do that by writing an AHK script. Just download [AHK](https://www.autohotkey.com/) and then create an .ahk file and write the code to run a specific application "in hidden mode" upon the script run. The script code would be something like this:
> (FOLLOW EACH AND EVERY SPACE PROPERLY AND FILL IN THE PARTS WITH <> NOTATIONS)
> 
> ```
> Run, <directory of your .exe file>, ,Hide
> WinWait, ahk_exe <<exact name of the exe file>.exe>
> ExitApp
> ```
> 
> After completing your script you can make it autorun everytime upon booting up your PC by putting its shortcut in the RUN=>shell:startup=>folder. Check [this](https://www.digitalcitizen.life/enable-apps-run-windows-10-startup/) if you don't know how to do this.
> 
> If you face any problems lemme know. Also if you want your EasyRP to work in a better way then check out this [repo!](https://github.com/bBSempai/EasyRP)

Hi @bBSempai, I want the console window to be hidden, I did try your ahk script but it doesn't work, no errors, I just double click on the .ahk file and it does nothing.
